scrapers/Iafd.yml

```diff
@@ -10,6 +15,18 @@ performerByURL:
     scraper: performerScraper
 
 xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1/text()
+      Date: 
+        selector: //div[@class="col-xs-12 col-sm-3"]//p[text() = 'Release Date']/following-sibling::p[1]/text()
+        parseDate: January 2, 2006
+      Details:  //div[@id="synopsis"]/div[@class="padded-panel"]//text()
+      Studio:
+        Name: //div[@class="col-xs-12 col-sm-3"]//p[text() = 'Studio']/following-sibling::p[1]/text()
```

Comment:
the studio doesn't work
you have to either do
`//div[@class="col-xs-12 col-sm-3"]//p[text() = 'Studio']/following-sibling::p[1]/a/text()`
since the text belonged to the `anchor` and not the `p`
or
`//div[@class="col-xs-12 col-sm-3"]//p[text() = 'Studio']/following-sibling::p[1]`
which would leave it to stash to evaluate the node text of the `p`  and get everything inside he `p`  including the `anchor `
i would recommend the 2nd option